mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -341,11 +341,12 @@ class GBTClassificationModel private[ml](
    * The importance vector is normalized to sum to 1. This method is suggested by Hastie et al.
```

Comment:
This comment is needed to update.

Reply:
no, it is still valid. The final vector is still normalized to 1.

Reply:
don't you skip normalization of importance vector?

Reply:
oh, I see. The normalization mentioned here is for total importance.

Reply:
the normalization of the importance vector for each tree, but then at the end the vector is still normalized. To simplify in a diagram, before the PR it was:
`tree importance` -> `normalization` -> `sum` -> `normalization`
now it is
`tree importance` -> `sum` -> `normalization`
So the final result is still normalized.